src/componentLibrary/theme/theme.ts

```diff
@@ -57,14 +57,19 @@ const theme = {
     lg: 'var(--rounded-lg)',
```

Comment:
What convention shall we follow for file names? So far, we have been following kebab-case. `componentLibrary` is camelCase though.